### PR TITLE
adding login with google

### PR DESCRIPTION
**Pull request by gonziesc**
## Summary
- Adding google login

## Trello card
https://trello.com/c/PMzKmrWr/15-integrar-el-blog-con-google-login

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -3,4 +3,6 @@ MAILER_PASSWORD=f1817eab4f1cc5
 MAILER_USER_NAME=ccd3b4d9d5e93a
 MAILER_ADDRESS=smtp.mailtrap.io
 MAILER_PORT=2525
-MAILER_AUTHENTICATION=cram_md5
\ No newline at end of file
+MAILER_AUTHENTICATION=cram_md5
```
Comment (hdf1986) [inline .env]: I've just noted this.... you are pushing the .env file! we need to remove this from the history asap

File: app/controllers/omniauth_callbacks_controller.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class OmniauthCallbacksController < Devise::OmniauthCallbacksController
+  # rubocop:disable Metrics/AbcSize
```
Comment (hdf1986) [inline app/controllers/omniauth_callbacks_controller.rb]: Remember to re-enable rubocop after doing this kind of things :)

File: config/initializers/devise.rb [hunk 3]
```diff
@@ -274,4 +274,6 @@
   # When using OmniAuth, Devise cannot automatically set OmniAuth path,
   # so you need to do it manually. For the users scope, it would be:
   # config.omniauth_path_prefix = '/my_engine/users/auth'
+
+  config.omniauth :google_oauth2, ENV['GOOGLE_CLIENT_ID'], ENV['GOOGLE_CLIENT_SECRET'], {}
```
Comment (hdf1986) [inline config/initializers/devise.rb]: Try using secrets instead of just linking to env, check at your config/secrets.yml for an example

**Review by hdf1996 — CHANGES_REQUESTED** [review]

File: .env (hunk 1 above)
Comment (gonziesc) [reply .env]: WHAT

File: config/secrets.yml [hunk 4]
```diff
@@ -25,6 +25,8 @@ development:
   mailer_address: <%= ENV["MAILER_ADDRESS"] %>
   mailer_port: <%= ENV["MAILER_PORT"] %>
   mailer_authentication: <%= ENV["MAILER_AUTHENTICATION"] %>
+  google_client_id: <%= ENV['GOOGLE_CLIENT_ID'] %>
```
Comment (hdf1986) [inline config/secrets.yml]: Don't forget about production and the testing environment :P

**Review by hdf1996 — CHANGES_REQUESTED** [review]

File: config/secrets.yml (hunk 4 above)
Comment (gonziesc) [reply config/secrets.yml]: noooo again lol

**Review by hdf1996 — APPROVED** [review]

**Pull request merged**